### PR TITLE
Use BColumnListView

### LINKED ISSUE #30: Switch from libsanta to Haiku API
It looks like only ColumnListView is used from libsanta.

### PR DESCRIPTION
**Pull request by AdrianArroyoCalle**
This should remove the libsanta dependency

GCI: https://codein.withgoogle.com/dashboard/task-instances/6260243134676992/?sp-page=1

Closes #30 and #31

### CODE REVIEW

File: bepdf/beos/AttachmentView.cpp [hunk 1]
```diff
@@ -189,22 +158,22 @@ AttachmentView::AttachmentView(::ToolTip* tooltip, BRect rect, GlobalSettings *s
...
-	AddChild(container);	
+		B_WILL_DRAW | B_FRAME_EVENTS | B_NAVIGABLE,
+		B_FANCY_BORDER,
+		true);
+	mList->AddColumn(new BStringColumn(TRANSLATE("File Name"), settings->GetAttachmentFileNameColumnWidth(), 10, 1000, true),0);
+	mList->AddColumn(new BStringColumn(TRANSLATE("Description"), settings->GetAttachmentDescriptionColumnWidth(), 10, 1000, true),1);
+	//mList->AddColumn(new CLVColumn(TRANSLATE("File Name"), settings->GetAttachmentFileNameColumnWidth(), CLV_SORT_KEYABLE | CLV_TELL_ITEMS_WIDTH));
+	//mList->AddColumn(new CLVColumn(TRANSLATE("Description"), settings->GetAttachmentDescriptionColumnWidth(), CLV_SORT_KEYABLE | CLV_TELL_ITEMS_WIDTH));
+	//mList->SetSortFunction(AttachmentItem::Compare);
+	//AddChild(container);
```
Comment (waddlesplash) [inline bepdf/beos/AttachmentView.cpp]: If this code is not being used anymore, remove it.

File: bepdf/beos/AttachmentView.h [hunk 2]
```diff
@@ -4,6 +4,7 @@
  * 	 Copyright (C) 1998-2000 Hubert Figuiere.
  * 	 Copyright (C) 2000-2011 Michael Pfeiffer.
  * 	 Copyright (C) 2013 waddlesplash.
+ *   Copyright (C) 2016 Adrián Arroyo Calle
```
Comment (waddlesplash) [inline bepdf/beos/AttachmentView.h]: Make this indented the same way the rest of the copyrights are.

File: bepdf/beos/Attachments.cpp [hunk 3]
```diff
@@ -59,9 +59,11 @@ static int sortByName(const void* a, const void* b) {
 	return strcmp(sa, sb);  
 }
 
-void Attachments::Replace(BListView *view) {
-	MakeEmpty(view);
+void Attachments::Replace(BColumnListView *view) {
+	view->Clear();
 	mAttachments.SortItems(sortByName);
-	view->AddList(&mAttachments);
+	for(int i=0;i<mAttachments.CountItems();i++) {
```
Comment (waddlesplash) [inline bepdf/beos/Attachments.cpp]: Properly format this according to Haiku coding style.

File: bepdf/beos/FileInfoWindow.cpp [hunk 4]
```diff
@@ -266,7 +266,7 @@ void FontItem::DrawItemColumn(BView* owner, BRect item_column_rect, int32 column
 
 void FontItem::Update(BView *owner, const BFont *font)
 {
-	CLVListItem::Update(owner,font);
+	//BRow::Update(owner,font);
```
Comment (waddlesplash) [inline bepdf/beos/FileInfoWindow.cpp]: Same here -- delete this if not needed.

**Comment by AdrianArroyoCalle** [comment]
Updated

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**